packages/shared/styles/components/SfButton.scss

```diff
@@ -1,4 +1,5 @@
 @import '../variables';
+@import '../helpers';
```

Comment:
Can't include helpers here, it's imported on global scss file and would be duplicated..